lms/templates/courseware/courseware.html

```diff
@@ -61,7 +61,10 @@
 <%static:css group='style-student-notes'/>
 % endif
 
-<%include file="../discussion/_js_head_dependencies.html" />
```

Comment:
Just FYI, courseware_chromeless is used to render XBlocks on mobile. You can manually see how this works by hitting this URL:

http://pr13483.sandbox.opencraft.hosting/xblock/block-v1:edX+DemoX+Demo_Course+type@vertical+block@vertical_3888db0bc286